packages/patterns-react/src/patterns/ListSection/ListSection.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import classNames from 'classnames';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { LISTSECTION } from '../../internal/FeatureFlags';
+import ListSectionGroup from './ListSectionGroup';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * List Section pattern
+ *
+ * @param {object} props props object
+ * @param {string} props.title List section title
+ * @param {string} props.copy List section  short copy to support the title
+ * @param {object} props.listGroup variation of the List section standard, standard with jump link and standard with card link
+ * @returns {object} JSX Object
+ */
+const ListSection = ({ title, copy, listGroup }) =>
+  featureFlag(
+    LISTSECTION,
+    <section
+      data-autoid={`${stablePrefix}--listsection`}
+      className={classNames(`${prefix}--listsection`)}>
+      <div className={`${prefix}--listsection__container`}>
+        <div className={`${prefix}--listsection__row`}>
+          <div className={`${prefix}--listsection__col`}>
+            <h3 className={`${prefix}--listsection__title`}>{title}</h3>
```

Comment:
@jeffchew Based on my understanding that's not the case anymore since HTML5. The new section elements like `section`, `article`, `header`, `footer`, `aside`, `nav` are self contained. It's more about semantics. (e.g. "For this `section`/`article` this is the main/h1 title")

https://www.suvaance.com/microblog/how-google-considers-multiple-h1-tags-in-one-page/
https://webdesign.tutsplus.com/articles/the-truth-about-multiple-h1-tags-in-the-html5-era--webdesign-16824
https://youtu.be/GIn5qJKU8VM
https://www.youtube.com/watch?v=WsgrSxCmMbM
https://html.spec.whatwg.org/multipage/sections.html#the-h1,-h2,-h3,-h4,-h5,-and-h6-elements

W3c might have an alternative viewpoint on it. Which might be causing some possible divisiveness on the topic.
https://www.w3.org/blog/2019/05/w3c-and-whatwg-to-work-together-to-advance-the-open-web-platform/
With that said they consider it valid in their [validation checker](https://validator.w3.org/).

Reply:
I think it also makes sense as we continue to make progress towards component-driven web pages. From a design system like ours, it's almost impossible to predict where a component might live in the hierarchy of the page.

Reply:
@photodow Also remember, we're not just targeting Google crawlers.

Reply:
And the common theme I have seen in all of the recent SEO articles (in the past year or less) is that `h1` is the page title, and `h2` is the content section title. Yet another article that iterates this:
https://seranking.com/blog/header-tags/

I have found a sea of them that say pretty much the same thing (then trying to sell their SEO services, lol).

Reply:
Those all appear to be opinion based articles. I was trying to stick to the sources. I know there are more than just Google search engines out there. I also understand the reasoning behind the single `h1`. There are alternative opinion based articles supporting the above too. Ultimately it boils down to different ways of thinking. Are we designing for a machine, or are we designing for humanss. What I'm seeing out there is that even when designing for the machine the benefit of a single `h1` while maybe common practice is minimal. You make the call I can only share what I know. 🤷‍♂️

Reply:
If by machines you mean web crawlers, SEO is an important factor here. How a screen reader breaks down the page by heading titles, it still achieves that with a single `h1` and `h2`'s for multiple content sections on the page. I think what SEO folks are saying makes sense to me and doesn't impact the user at the end of the day. Again, we can pull in our own internal SEO SMEs to weigh in.

cc: @wonilsuhibm

Reply:
Sounds like a game plan. Thanks Jeff.